frappe_geo_restrictions/utils/cache_invalidation.py

```diff
@@ -0,0 +1,46 @@
+import frappe
```

Comment:
We NEED to test this end to end as cache is used in most of the platform. Maybe we can add test cases?